src/show/pages/public/LogIn/index.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { MdOutlineAccountBox } from 'react-icons/md';
+
+import Paths from 'process/routes/paths';
+import { logIn } from 'process/slices/userSlice';
+
+import { Button, TextField } from 'show/components';
+
+const LogIn = () => {
+  const [client, setClient] = useState({
+    email: '',
+    password: ''
+  });
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const Submit = e => {
+    e.preventDefault();
+    dispatch(logIn({ email: client.email, password: client.password }));
```

Comment:
```suggestion
    dispatch(logIn(client));
```
@iamdavidpetric neither this one?

Reply:
done